### ISSUE TITLE
Django 1.11 support

### ISSUE THREAD

**Issue by georgesamy** [inonit/drf-haystack#113]
First of all great app, good job guys. Second of all, do you plan to support Django v1.11?

**Comment by rhblind** [comment]
Thank you for the kind words!
Yes, absolutely. The goal is to support all officially supported Django versions, however I'm *really* short on time these days!
If you want to help us, please feel free to send PRs =)

**Comment by rhblind** [comment]
Hey,
I got to do some work yesterday, and will likely have a new version out in a few days which supports Django1.11.

**Comment by georgesamy** [comment]
Hey, the last week was full.. have you worked on it? do you still need help?

**Comment by rhblind** [comment]
Hey, yes I've done some work on it in the [develop](https://github.com/inonit/drf-haystack/tree/develop) branch.
Only thing left is to make the [test suite](https://travis-ci.org/inonit/drf-haystack) pass for Django1.8. 
If you got the time, we're not far from a new release :)

**Comment by cdrandin** [comment]
I am unsure where to put this issue at, but I wanted to use this software and I was using Django 1.11. I pull off the development branch. All worked pretty well, except I encountered this error `AttributeError: 'SearchQuerySet' object has no attribute 'model'`.

Figured I post it in case anyone else may have this problem.

I followed the example [here](https://drf-haystack.readthedocs.io/en/latest/01_intro.html#examples)

I was able to find a work around by doing the following :

```
class <model>SearchView(HaystackViewSet):

    # `index_models` is an optional list of which models you would like to include
    # in the search result. You might have several models indexed, and this provides
    # a way to filter out those of no interest for this particular view.
    # (Translates to `SearchQuerySet().models(*index_models)` behind the scenes.
    index_models = [<model>]
    serializer_class = <model>Serializer

    def get_queryset(self, *args, **kwargs):
        queryset = super(HaystackViewSet, self).get_queryset(*args, **kwargs)
        queryset.model = <model>
        return queryset
``` 

Here is my `search_indexes.py` just in case:

```
class <model>Index(indexes.SearchIndex, indexes.Indexable):
    text = indexes.CharField(document=True, use_template=True)
    description = indexes.CharField(model_attr='description')
    autocomplete = indexes.EdgeNgramField()

    @staticmethod
    def prepare_autocomplete(obj):
        return obj.description

    def get_model(self):
        return <model>
```

**Comment by rhblind** [comment]
@cdrandin would you be able to open this as a separate issue, and post the error traceback you receive?

**Comment by cdrandin** [comment]
@rhblind sure thing.

**Comment by amadornimbis** [comment]
Any word on possible Django 1.11 support?

**Comment by rhblind** [comment]
Released v1.7.1rc2 now. It depends on django-haystack `>=v2.7dev0`, so we'll go stable once they do.
